cmake/modules/SwiftSource.cmake

```diff
@@ -269,7 +269,7 @@ function(_compile_swift_files
   endif()
 
   # Force swift 3 compatibility mode for Standard Library and overlay.
-  if (SWIFTFILE_IS_STDLIB_CORE OR SWIFTFILE_IS_SDK_OVERLAY)
+  if (SWIFTFILE_IS_STDLIB OR SWIFTFILE_IS_SDK_OVERLAY)
```

Comment:
Which files don't fit into either of these categories? Things like StdlibUnittest?